src/Tribe/Commerce/Warnings.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Conditional Warnings on Ticket Features
+ *
+ * Class Tribe__Tickets__Commerce__Warnings
+ *
+ * @since TBD
+ */
```

Comment:
```suggestion
/**
 * Conditional Warnings on Ticket Features.
 */

namespace Tribe\Tickets\Commerce;

/**
 * Warnings handling class.
 *
 * @since TBD
 */
```